packages/react/src/components/Notification/stories/InlineNotification.stories.js

```diff
@@ -23,6 +23,7 @@ export default {
     kind: 'error',
     lowContrast: false,
     hideCloseButton: false,
+    ['aria-label']: 'closes notification',
```

Comment:
should we remove the `ariaLabel` arg here? seems weird to have both. Same comment for the other notifications

Reply:
@francinelucca added the changes